x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/data_streams/flyout/container.tsx

```diff
@@ -98,6 +99,12 @@ export const DataStreamReindexFlyout: React.FunctionComponent<Props> = ({
   }, [loadDataStreamMetadata, switchFlyoutStep, flyoutStep]);
   useMemo(() => switchFlyoutStep(), [switchFlyoutStep]);
 
+  useEffect(() => {
```

Comment:
im a bit worried about the introduction of this effect, I think it might be better to trigger it only based on explicit user action